libs/parser-sql/src/postgresParser.ts

```
@@ -0,0 +1,1652 @@
+const Returning = createToken({name: 'Returning', pattern: /\bRETURNING\b/i, longer_alt: Identifier})
+const Right = createToken({name: 'Right', pattern: /\bRIGHT\b/i, longer_alt: Identifier})
+const Row = createToken({name: 'Row', pattern: /\bROW\b/i, longer_alt: Identifier})
+const Rows = createToken({name: 'Rows', pattern: /\bROWS\b/i, longer_alt: Identifier})
+const Schema = createToken({name: 'Schema', pattern: /\bSCHEMA\b/i, longer_alt: Identifier})
+const Select = createToken({name: 'Select', pattern: /\bSELECT\b/i, longer_alt: Identifier})
+const Sequence = createToken({name: 'Sequence', pattern: /\bSEQUENCE\b/i, longer_alt: Identifier})
+const Serializable = createToken({name: 'Serializable', pattern: /\bSERIALIZABLE\b/i, longer_alt: Identifier})
+const Session = createToken({name: 'Session', pattern: /\bSESSION\b/i, longer_alt: Identifier})
+const SetDefault = createToken({name: 'SetDefault', pattern: /\bSET\s+DEFAULT\b/i})
+const SetNull = createToken({name: 'SetNull', pattern: /\bSET\s+NULL\b/i})
+const Set = createToken({name: 'Set', pattern: /\bSET\b/i, longer_alt: Identifier})
```

Biome

[error] Do not shadow the global "Set" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)